app/controllers/websites_controller.rb

```diff
@@ -1,57 +1,33 @@
 class WebsitesController < ApplicationController
-	  require 'open-uri'
-	  require 'nokogiri'
-    require 'net/http'
 
   def new
-  	@website = Website.new
-
+    @website = Website.new
   end
 
   def create
-  	@website = Website.new(website_params)
-
-    @h1_title = ""
-    @paragraph = ""
-    @final_content = " "
-
-    url = params[:website][:url]
-
-    data = Nokogiri::HTML(open(url))
-    contents = data.css("html")
-    contents.each do |content|
-      @h1_title = content.css("h1").text.strip
-      @paragraph = content.css("p").text.strip
-    end
-
-    @final_content = @h1_title + @paragraph
-
-    array = Array.new
-    array = @final_content.split(" ")
-
-    @website.content = array
+    @website = Website.new(website_params)
 
+    @website.content = ContentExtractor.new(@website.url).extract
 
   	if @website.save
   		redirect_to new_website_path
+    else
+      render :action => 'index'
```

Comment:
Maybe you can add some sort of flash message here + check indentation.

Pro tip: 
In vim you can use `gg=G` keystroke to auto indent whole file. :)